c_glib/arrow-glib/composite-array.cpp

```diff
@@ -2044,4 +2046,117 @@ garrow_run_end_encoded_array_find_physical_length(GArrowRunEndEncodedArray *arra
   return arrow_run_end_encoded_array->FindPhysicalLength();
 }
 
+G_DEFINE_TYPE(GArrowFixedSizeListArray, garrow_fixed_size_list_array, GARROW_TYPE_ARRAY)
+
+static void
+garrow_fixed_size_list_array_init(GArrowFixedSizeListArray *object)
+{
+}
+
+static void
+garrow_fixed_size_list_array_class_init(GArrowFixedSizeListArrayClass *klass)
+{
+}
+
+/**
+ * garrow_fixed_size_list_array_new_list_size
+ * @value_array: The value of the fixed list size array.
+ * @list_size: The list size of the fixed list size array.
+ * @null_bitmap: (nullable): The bitmap that shows null elements. The
+ *   N-th element is null when the N-th bit is 0, not null otherwise.
+ *   If the array has no null elements, the bitmap must be %NULL and
+ *   @n_nulls is 0.
+ * @n_nulls: The number of null elements. If -1 is specified, the
+ *   number of nulls are computed from @null_bitmap.
+ * @error: (nullable): Return location for a #GError or %NULL.
+ *
+ * Returns: (nullable): A newly created #GArrowFixedSizeListArray
+ *   or %NULL on error.
+ *
+ * Since: 22.0.0
+ */
+GArrowFixedSizeListArray *
+garrow_fixed_size_list_array_new_list_size(GArrowArray *value_array,
+                                           gint32 list_size,
+                                           GArrowBuffer *null_bitmap,
+                                           gint64 n_nulls,
+                                           GError **error)
+{
+  std::shared_ptr<arrow::Array> arrow_value_array = garrow_array_get_raw(value_array);
+  auto arrow_null_bitmap = garrow_buffer_get_raw(null_bitmap);
+
+  auto arrow_fixed_size_list_array_result =
+    arrow::FixedSizeListArray::FromArrays(arrow_value_array,
```

Comment:
I use `arrow::FixedSizeListArray::FromArrays`. It's returns `Result<std::shared_ptr<Array>>`

Should I use another method? (ex. `FixedSizeListArray::Make`?)

Reply:
`FromArrays()` is OK because `Make()` doesn't exist`.

Reply:
OK Thanks.